styles.css

```diff
@@ -100,18 +104,297 @@ h1 {
   color: #f55800;
 }
 
+.works {
+  background-color: #e5e5e5;
+  padding: 2rem 10rem;
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+}
+
+h3 {
+  font-family: "Crete Round", serif;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  color: #091e42;
+  text-align: center;
+}
+
+.header-content {
+  text-align: left;
+  margin: 0;
+}
+
+p {
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 32px;
+  color: #f4f5f7;
+}
+
+.work-title {
+  font-family: "Crete Round", serif;
+  font-weight: 400;
+  font-size: 40px;
+  line-height: 52px;
+  color: #091e42;
+  align-items: center;
+  text-align: center;
+  width: 100%;
+  flex: 1;
+}
+
+.work-line-title {
+  border: 1px solid #212121;
+  width: 90%;
+  margin: auto;
+  flex: 3;
+}
+
+.works-title {
+  align-items: center;
+  padding-bottom: 2rem;
+  display: flex;
+}
+
+.class-works {
+  display: flex;
+}
+
+.initial-section {
+  display: flex;
+  width: 100%;
+  margin: auto;
+}
+
+.principal-image {
+  margin: auto;
+}
+
+.content {
+  padding: 0 2rem;
+}
+
+.content p {
+  color: #091e42;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+}
+
+.tag-container {
+  display: flex;
+  align-items: center;
+  padding-bottom: 2rem;
+}
+
+.tag {
+  border: 1px solid #8993a4;
+  box-sizing: border-box;
+  font-size: 15px;
+  line-height: 20px;
+  padding: 10px 12px;
+  color: #091e42;
+  margin: 5px;
+}
+
+.button {
+  background-color: #f55800;
+  color: #fff;
+  padding: 12px;
+  font-family: Inter, sans-serif;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  border: none;
+}
+
+.button:hover {
+  background-color: #e97f42;
+}
```

Comment:
- [ ] Please, consider that the hover should be as follows, as in the Figma template:
```
.button:hover {
  background-color: #FFF5E1;
  color: #F55800;
  border: 1px solid #F55800;
  box-sizing: border-box;
}
```